src/github.com/couchbase/sync_gateway/db/kv_change_index_writer.go

```diff
@@ -173,15 +179,29 @@ func (k *kvChangeIndexWriter) indexPending() {
 	// Generic channelStorage for log entry storage (if needed)
 	channelStorage := NewChannelStorage(k.indexWriteBucket, "", indexPartitions)
 
+	indexRetryCount := 0
+	maxRetries := 10
```

Comment:
I wonder if we should make this larger since the process will essentially crash if this gets exhausted .. what's the total time it will retry for with 10 retries and starting at 5 ms?